### PR TITLE
Added rest of Terminal Directive Cards

### PR DESCRIPTION
**Pull request by NoahTheDuke**
Added all of the new cards spoiled from [here](https://imgur.com/a/jimfv).

### CODE REVIEW

File: pack/td.json [hunk 1]
```diff
@@ -54,6 +54,24 @@
         "uniqueness": false
     },
     {
+        "code": "13004",
+        "cost": 2,
+        "deck_limit": 3,
+        "faction_code": "criminal",
+        "faction_cost": 3,
+        "flavor": "Flooding the system with synchronized request messages makes it unresponsive to legitimate requests.",
+        "illustrator": "Alexandr Eichev",
```
Comment (lostie) [inline pack/td.json]: **[typo]** The illustrator's surname is **_Elichev_**.

File: pack/td.json [hunk 2]
```diff
@@ -434,6 +528,40 @@
...
+        "code": "13030",
+        "deck_limit": 3,
+        "faction_code": "haas-bioroid",
+        "illustrator": "Matt Zeilinger",
+        "keywords": "Initiative",
+        "pack_code": "td",
+        "position": 30,
+        "quantity": 3,
+        "side_code": "corp",
+        "text": "Place 2 agenda counters on Elective Upgrade when you score it.\n[click], <strong>Hosted agenda counter</strong>: Gain [click][click]. Use this ability only once per turn.",
```
Comment (lostie) [inline pack/td.json]: **[Minor Typo/nit pick]** '**_hosted_** agenda counter...' is typed lowercase in the card's text.

File: pack/td.json [hunk 3]
```diff
@@ -434,6 +528,40 @@
...
+        "type_code": "agenda",
+        "uniqueness": false
+    },
+    {
+        "advancement_cost": 4,
+        "agenda_points": 2,
+        "code": "13031",
+        "deck_limit": 3,
+        "faction_code": "haas-bioroid",
+        "illustrator": "Clarck Huggins",
```
Comment (lostie) [inline pack/td.json]: **[type]** Illustrator's first name is **_Clark_**.

File: pack/td.json [hunk 4]
```diff
@@ -489,6 +617,80 @@
         "uniqueness": false
     },
     {
+        "code": "13035",
+        "cost": 2,
+        "deck_limit": 3,
+        "faction_code": "haas-bioroid",
+        "faction_cost": 3,
+        "flavor": "\"Strictly speaking, it isn't illegal to use an interrogation simulator as the blueprint for ICE.\" - Mason Bellamy",
+        "illustrator": "Andreas Zafiratos",
```
Comment (lostie) [inline pack/td.json]: I'm not sure if this a typo at all, the card has the illustrator's surname as _**Zafeiratos**_, but a quick search on [deviant art](http://albino-z.deviantart.com/) points to the name you have inputed. 😕

File: pack/td.json [hunk 5]
```diff
@@ -489,6 +617,80 @@
...
+        "type_code": "ice",
+        "uniqueness": false
+    },
+    {
+        "code": "13036",
+        "cost": 7,
+        "deck_limit": 3,
+        "faction_code": "haas-bioroid",
+        "faction_cost": 2,
+        "illustrator": "Andreas Zafiratos",
```
Comment (lostie) [inline pack/td.json]: [Ditto](https://github.com/zaroth/netrunner-cards-json/pull/210/files#r113030963)

File: pack/td.json [hunk 6]
```diff
@@ -489,6 +617,80 @@
...
+        "type_code": "ice",
+        "uniqueness": false
+    },
+    {
+        "code": "13037",
+        "cost": 5,
+        "deck_limit": 3,
+        "faction_code": "haas-bioroid",
+        "faction_cost": 1,
+        "flavor": "\"The Warp and West were hypnotic...but they didn't really seem to do anything to keep me out. But I never seemed to find anything interesting either.\" - Reaver",
```
Comment (lostie) [inline pack/td.json]: **[typo]** 'The Warp and **_Weft_**...'

File: pack/td.json (hunk 4 above)
Comment (Alsciende) [inline pack/td.json]: **Andreas Zafiratos** is already referenced as an illustrator in nrdb. Better to correct FFG's mistake.

File: pack/td.json [hunk 7]
```diff
@@ -489,6 +617,80 @@
...
+        "type_code": "ice",
+        "uniqueness": false
+    },
+    {
+        "code": "13038",
+        "cost": 6,
+        "deck_limit": 3,
+        "faction_code": "haas-bioroid",
+        "faction_cost": 4,
+        "flavor": "If there were hidden partitions on the server, I'd know; I'm the head of this department!",
```
Comment (lostie) [inline pack/td.json]: This flavour text is itself in quotes, so needs `"\"...\""`

File: pack/td.json [hunk 8]
```diff
@@ -526,6 +747,75 @@
...
+        "code": "13042",
+        "deck_limit": 3,
+        "faction_code": "weyland-consortium",
+        "illustrator": "Ed Mattinian",
+        "keywords": "Security",
+        "pack_code": "td",
+        "position": 42,
+        "quantity": 3,
+        "side_code": "corp",
+        "text": "Place 1 agenda counter on Armored Servers when you score it.\n<strong>Hosted agenda counter</strong>: For the remainder of this run, the runner must trash a card from his or her grip as an additional cost to jack out or break a subroutine. Use this ability only during a run.",
```
Comment (lostie) [inline pack/td.json]: FFG's templating is horrible and inconsistent :/

Here **_Runner_** is capitalized.

File: pack/td.json [hunk 9]
```diff
@@ -526,6 +747,75 @@
...
+        "title": "Successful Field Test",
```
Comment (lostie) [inline pack/td.json]: **[typo]** Title should be _**Armored Servers**_

File: pack/td.json [hunk 10]
```diff
@@ -526,6 +747,75 @@
...
+        "type_code": "agenda",
+        "uniqueness": false
+    },
+    {
+        "advancement_cost": 3,
+        "agenda_points": 1,
+        "code": "13043",
+        "deck_limit": 3,
+        "faction_code": "weyland-consortium",
+        "flavor": "\"Anything to anyone...for the right price.\"",
```
Comment (lostie) [inline pack/td.json]: Missing a comma, 'Anything, to anyone...'

File: pack/td.json [hunk 11]
```diff
@@ -526,6 +747,75 @@
...
+        "type_code": "agenda",
+        "uniqueness": false
+    },
+    {
+        "advancement_cost": 5,
+        "agenda_points": 3,
+        "code": "13044",
+        "deck_limit": 3,
+        "faction_code": "weyland-consortium",
+        "illustrator": "Del Borevio",
```
Comment (lostie) [inline pack/td.json]: I believe the illustrator's name is **Del Borovic** : http://delborovic.tumblr.com/

File: pack/td.json (hunk 4 above)
Comment (lostie) [inline pack/td.json]: I agree, the number of templating issues and typos in the most recent packs is really not acceptable from a company like FFG, reveals lack of care and quality control.

**Comment by lostie** [comment]
Overall, LGTM 👍 , has a couple of typos that need sorting, most specially the one where the agenda's title is incorrect.

File: pack/td.json (hunk 4 above)
Comment (NoahTheDuke) [inline pack/td.json]: I'll fix the rest of them, then.

File: pack/td.json (hunk 11 above)
Comment (NoahTheDuke) [inline pack/td.json]: Thank you. I spent a solid minute trying to read the name.

**Comment by NoahTheDuke** [comment]
I'll fix the errors momentarily and re-up!

File: pack/td.json (hunk 6 above)
Comment (ANR-PeterPan) [inline pack/td.json]: and If Successful is missing on Holmegaard too

**Comment by Alsciende** [comment]
Thank you all!

File: pack/td.json [hunk 12]
```diff
@@ -489,6 +617,80 @@
...
+        "faction_cost": 3,
+        "flavor": "\"Strictly speaking, it isn't illegal to use an interrogation simulator as the blueprint for ICE.\" - Mason Bellamy",
+        "illustrator": "Andreas Zafiratos",
+        "keywords": "Code Gate - AP - Tracer",
+        "pack_code": "td",
+        "position": 35,
+        "quantity": 3,
+        "side_code": "corp",
+        "strength": 1,
+        "text": "[subroutine] <trace>trace 4</trace> Do 1 brain damage.",
```
Comment (lostie) [inline pack/td.json]: a very small thing I've noticed in the live site, the _**Trace**_ should be capitalised.

File: pack/td.json [hunk 13]
```diff
@@ -489,6 +617,80 @@
...
+        "faction_code": "haas-bioroid",
+        "faction_cost": 2,
+        "illustrator": "Andreas Zafiratos",
+        "keywords": "Sentry - Tracer - Destroyer",
+        "pack_code": "td",
+        "position": 36,
+        "quantity": 3,
+        "side_code": "corp",
+        "strength": 5,
+        "text": "[subroutine] <trace>trace 4</trace> The runner cannot access any cards during this run.\n[subroutine] Trash 1 <strong>icebreaker</strong>.",
```
Comment (lostie) [inline pack/td.json]: [Ditto, same here](https://github.com/zaroth/netrunner-cards-json/pull/210/files#r113071559)

Comment (Alsciende) [inline pack/td.json]: @lostie Thanks, I commited a fix

Comment (Alsciende) [inline pack/td.json]: @lostie we still need Hailstorm's strength

**Comment by NoahTheDuke** [comment]
Ah crap. Thanks for catching all these, @lostie.

**Pull request merged**